.github/workflows/release-please.yml

```diff
@@ -1,22 +1,28 @@
-name: Release Please
-
 on:
   push:
     branches:
       - main
-
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+name: release-please
 jobs:
-  release:
-    name: Run release-please
-    runs-on: ubuntu-latest
-
+  release-please:
+    runs-on: [ARM64, self-hosted, Linux]
```

Comment:
Any particular reason we're running this on ARM?

Reply:
I copied that from the action provided by central infra. Not sure what difference this would make.

Reply:
It just runs the action on our internal infrastructure (and using ARM nodes) instead of letting GitHub run it (by default on x86 nodes). Since this is a public-facing project I'd prefer to use the standard GitHub nodes as much as possible.

Reply:
ok! I'll revert that :)